Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml

```diff
@@ -1441,64 +1134,125 @@ script:
       description: Process display name.
       type: String
   - arguments:
-    - default: false
-      description: 'A free-text search term, will match applicable attributes (sub-string match). Note: A device''s physical addresses will only be matched if they start with the search term  (not if they contain the search term).'
-      isArray: false
+    - description: "A free-text search term, will match applicable attributes (sub-string\
+        \ match). Note: A device's physical addresses will only be matched if they\
+        \ start with the search term  (not if they contain the search term)."
       name: query
-      required: false
-      secret: false
-    - default: false
-      description: A comma-separated list of agents IDs to shutdown.
+    - description: A comma-separated list of agents IDs to shutdown.
       isArray: true
       name: agent_id
-      required: false
-      secret: false
-    - default: false
-      description: The ID of the network group.
-      isArray: false
+    - description: The ID of the network group.
       name: group_id
-      required: false
-      secret: false
-    deprecated: false
     description: Sends a shutdown command to all agents that match the input filter.
-    execution: false
     name: sentinelone-shutdown-agent
     outputs:
     - contextPath: SentinelOne.Agent.ID
       description: The ID of the agent that was shutdown.
       type: String
   - arguments:
-    - default: false
-      description: 'A free-text search term, will match applicable attributes (sub-string match). Note: A device''s physical addresses will only be matched if they start with the search term  (not if they contain the search term).'
-      isArray: false
+    - description: "A free-text search term, will match applicable attributes (sub-string\
+        \ match). Note: A device's physical addresses will only be matched if they\
+        \ start with the search term  (not if they contain the search term)."
       name: query
-      required: false
-      secret: false
-    - default: false
-      description: A comma-separated list of agents IDs to shutdown.
-      isArray: false
+    - description: A comma-separated list of agents IDs to shutdown.
       name: agent_id
-      required: false
-      secret: false
-    - default: false
-      description: The ID of the network group.
-      isArray: false
+    - description: The ID of the network group.
       name: group_id
-      required: false
-      secret: false
-    deprecated: false
     description: Sends an uninstall command to all agents that match the input filter.
-    execution: false
     name: sentinelone-uninstall-agent
```

Comment:
are the changes from here above were intentional? did you intend to do any change on the configuration? 
If not then please revert all of this back to the origin